<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gerenciaChamados</class>
 <widget class="QDialog" name="gerenciaChamados">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>698</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>599</width>
    <height>698</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gerenciamento de Chamados</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>14</y>
     <width>521</width>
     <height>633</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineID"/>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Matriz RACI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="abreRACI">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Abre RACI</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>SLA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineSLA">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboStatus">
        <item>
         <property name="text">
          <string>Aberto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Atribuído</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Em Execução</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Concluído</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Reincidente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkReincidente">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Aberto em</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="timeAberto">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Ativos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="abreDisp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Abre Ativos</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Concluído em</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="timeConcluido"/>
      </item>
     </layout>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Acompanhamentos</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="adicionaAcompanhamento">
      <property name="text">
       <string>Adiciona Acompanuhamento</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Descrição</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="textDescricao">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="1">
     <widget class="QTableView" name="tableAcompanha">
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLineEdit" name="lineAcompanhamento"/>
    </item>
    <item row="11" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Salvar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Editar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
